starctl-service-rest

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+if type -p java >/dev/null; then
+    _java=java
+elif [[ -n "$JAVA_HOME" ]] && [[ -x "$JAVA_HOME/bin/java" ]];  then
+    _java="$JAVA_HOME/bin/java"
+else
+    echo "\$JAVA_HOME not found"
+    exit 1
+fi
+
+if [[ "$_java" ]]; then
+    jvmver=$("$_java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
+    JVM_VERSION=${jvmver%_*}
+    JVM_PATCH_VERSION=${jvmver#*_}
+
+    if [ "$JVM_VERSION" \< "1.8" ] ; then
+        echo "Stargate requires Java 8u40 or later."
```

Comment:
I assume this comes from existing codebase, but I wonder if there's much point... 1.8.40 is ancient as-is. Could reduce less useful checking.
(specifically thinking that looking for 1.8+ is fine, but patch level is bit arbitrary)